media/css/mozorg/contribute/contribute-common.less

```diff
@@ -0,0 +1,225 @@
+        width: @widthMobileLandscape - (@gridGutterWidth * 2);
+    }
+}
+
+// @Navigation
+.contribute-masthead {
+    position: absolute;
+    top: 0;
+    z-index: 1;
+    background: @mozillaRed;
+    padding: 40px 0 10px;
+    width: 100%;
+    min-height: 40px;
+    border-bottom: 1px solid darken(@mozillaRed, 5%);
+    .section-dark;
+    .clearfix;
+
+    @media only screen and (max-width: @breakTablet) {
+        min-height: 0;
+    }
+}
+
+.contribute-nav {
+    .span(10);
+
+    @media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
+        .span_narrow(10);
+    }
+
+    @media only screen and (max-width: @breakTablet) {
+        width: auto;
+        float: none;
+        position: absolute;
+        left: -10px;
+        top: 0;
+
+        #contribute-nav-menu {
+            display: none;
+            width: 160px;
+            background: @mozillaRed;
+            padding: 0 5px 5px;
+            border-radius: 6px;
+            left: 0;
+
+            li {
+                width: auto;
+                margin: 5px 0 0;
+                float: none;
+                display: block;
+            }
+
+            .nav-cta {
+                padding-bottom: 8px;
+            }
+        }
+    }
+
+    .toggle {
+        display: none;
+
+        @media only screen and (max-width: @breakTablet) {
+            position: absolute;
+            left: 0;
+            top: -30px;
+            display: block;
+            width: 32px;
+            height: 32px;
+            text-indent: 50px;
+            overflow: hidden;
+            cursor: pointer;
+            margin: 0;
+
+            &:after {
+                position: absolute;
+                top: 4px;
+                left: 0;
+                content: '';
+                background: transparent url('/media/img/contribute/signup/burger.svg') 0 0 no-repeat;
+                width: 30px;
+                height: 30px;
+            }
+        }
+    }
+
+    li {
+        .span(2);
+        display: block;
+        font-weight: bold;
+        line-height: 1.1;
+        list-style: none;
+        min-height: 40px;
+        position: relative;
+        text-align: center;
+        text-shadow: 0 1px 0 rgba(0, 0, 0, .1);
+        text-transform: uppercase;
+        vertical-align: bottom;
+
+        @media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
+            .span_narrow(2);
+        }
+    }
+
+    a {
+        border-radius: 3px;
+        box-sizing: border-box;
+        color: #e29fa0;
+        display: block;
+        padding: 10px;
+        width: 100%;
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        .transition(all .15s ease-in-out);
+
+        &:hover,
+        &:focus,
+        &:active {
+            background-color: lighten(@mozillaRed, 10%);
+            background-color: rgba(255, 255, 255, .1);
```

Comment:
Hmmm, this came from the old contribute2015 less file, wonder if that was done on purpose for some reason. Maybe the first is a fallback for IE?